lib/mix/tasks/nerves_hub.device.ex

```diff
@@ -114,28 +114,29 @@ defmodule Mix.Tasks.NervesHub.Device do
 
     show_api_endpoint()
     org = org(opts)
+    product = product(opts)
 
```

Comment:
In order to use this, you’ll also need to add `:product` to the `@switches`, correct? Otherwise this will just always default and never set by the user

Reply:
And if so, we should probably update these docs as well to state product is required

Reply:
I've updated these